### ISSUE TITLE
运行结果为Predicting... P: 1.0000  R: 0.0000  F: 0.0000

### ISSUE THREAD

**Issue by xugege0523** [PoWWoP/UOI-1705.00108#1]

**Comment by xugege0523** [comment]
with no external corpora  used for training the bidirectional language model, just using conll03 ,I got a strange result. And log information is:
Fitting...
Epoch 1/10
878/878 [==============================] - 186s 211ms/step - loss: nan - acc: nan - val_loss: nan - val_acc: nan
Epoch 2/10
878/878 [==============================] - 184s 209ms/step - loss: nan - acc: nan - val_loss: nan - val_acc: nan
why the loss are all nan??

Please help me ! Thanks a lot!

**Comment by CyberZHG** [comment]
I'm not able to reproduce the error. Are you using the latest versions of keras and keras-contrib?

**Comment by xugege0523** [comment]
@CyberZHG  I just run your code without changing.
Can you give me your definite versions of keras and keras-contrib ?
mine is keras=2.1.4;keras-contrib=0.0.2;
And can you show your correct log information ?
Thanks a lot!

**Comment by CyberZHG** [comment]
* Keras==2.2.4
* keras-contrib==2.0.8
